src/content/en/2020/resource-hints.md

```diff
@@ -217,87 +224,58 @@ To safely implement `dns-prefetch` and `preconnect` make sure to have them in se
+Lastly, running Lighthouse's "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
 
-{{ figure_markup(
-  caption="Pages that pass the preload Lighthouse audit",
-  content="84.6%",
-  classes="big-number",
-  sheets_gid="1047875076",
-  sql_file="lighthouse_preload.sql"
-) }}
-
-Running Lighthouse’s "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test, which is an astonishing result. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
-
-{# TODO - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
+{# TODO(authors/reviewers) - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532291496 #}
 
 ### Native Lazy Loading
 
-Now let’s celebrate the first year of the [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/) API, which at the time of publishing already has over [72%](https://caniuse.com/loading-lazy-attr) browser support. This new API can be used to defer the load of below-the-fold iframes and images on the page until the user scrolls near them. This can reduce data usage, memory usage, and helps speed up above-the-fold content. Opting-in to lazy load is as simple as adding `loading=lazy`  on `<iframe>` or `<img>` elements.
-
-{# TODO - revisit this sentence - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r533106799 #}
+Now let's celebrate the first year of the [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/) API, which at the time of publishing already has over [72%](https://caniuse.com/loading-lazy-attr) browser support. This new API can be used to defer the load of below-the-fold iframes and images on the page until the user scrolls near them. This can reduce data usage, memory usage, and helps speed up above-the-fold content. Opting-in to lazy load is as simple as adding `loading=lazy`  on `<iframe>` or `<img>` elements.
 
 {{ figure_markup(
-  caption="Percentage of pages using native lazy loading set to “lazy”",
-  content="3.87%",
+  caption="The percent of pages using native lazy loading.",
+  content="4.02%",
   classes="big-number",
   sheets_gid="2039808014",
   sql_file="native_lazy_loading_attrs.sql"
 ) }}
 
-{# TODO - revisit this figure - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532292106 #}
+{# TODO(authors/reviewers) - revisit this figure - Ref https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532292106 #}
```

Comment:
@Zizzamia

Reply:
I was looking more over https://github.com/HTTPArchive/almanac.httparchive.org/pull/1587#discussion_r532292106 and I am not super sure if the query we run does make a distinction between `iframe` or `img`. Maybe @rviscomi or @khempenius can help a bit more with it. cc @bazzadp